Generator/GeneratorInterface.php

```diff
@@ -16,9 +16,7 @@
 interface GeneratorInterface
 {
     /**
-     * @abstract
```

Comment:
Hmm not sure about that, the method is not declared as abstract.

But I'm wondering the utility of this tag... cc @rande.